src/test/java/delete_task_use_case/DeleteTaskControllerTest.java

```diff
@@ -0,0 +1,36 @@
+package delete_task_use_case;
+
+import data_access.DataAccessDay;
+import data_access.DataAccessEvent;
+import data_access.DataAccessTask;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class DeleteTaskControllerTest {
+    private final static DeleteTaskPresenter outputBoundary = new DeleteTaskPresenter();
```

Comment:
Ideally your outputBoundary variable would also be of type DeleteTaskOutputBoundary, although you would assign to it an instance of type DeleteTaskPresenter

Reply:
Added this change, as suggested.